inc/drivers/MicroBitQuadratureDecoder.h

```diff
@@ -0,0 +1,175 @@
+/*
+The MIT License (MIT)
+
+Copyright (c) 2016 British Broadcasting Corporation.
+This software is provided by Lancaster University by arrangement with the BBC.
```

Comment:
Oops copy-paste.

I don't really want to own copyright on anything; what's the proper default copyright notice to put on new files?